redi-connect-front/src/pages/front/signup/signUpFormFactory.tsx

```diff
@@ -69,15 +69,15 @@ export interface SignUpFormValues {
 }
 
 const initialValues: SignUpFormValues = {
-  formType: 'public-sign-up-mentee-pending-review',
-  gaveGdprConsentAt: '',
-  username: '',
-  password: '',
-  passwordConfirm: '',
-  firstName: '',
-  lastName: '',
+  formType: "public-sign-up-mentee-pending-review",
+  gaveGdprConsentAt: "",
+  username: "",
+  password: "",
+  passwordConfirm: "",
+  firstName: "",
+  lastName: "",
   agreesWithCodeOfConduct: false,
-  mentee_currentlyEnrolledInCourse: '',
+  mentee_currentlyEnrolledInCourse: "",
 };
 
 export const buildSignUpForm = (
```

Comment:
Awesome! Very pleasing to see the old form going out the window and being replaced by our beautiful new bulma components :D